content/troubleshooting/java/installation/exec-helper-troubleshooting.md

```diff
@@ -2,39 +2,45 @@
 
-## Issue: I removed the libcontrast_universal_installer.so libraries, and now my system errors on every command I run. 
+## Issue: I removed the libcontrast_universal_installer.so libraries, and now my system errors on every command I run 
 
-The host emits these errors because the \*nix system has been globally configured to load the *libcontrast_universal_installer.so* library for every binary on the system to determine if it should inject the Java agent into it. Since the library was manually removed, the global configuration of dynamic loader is still referencing it and is failing/warning when it can't find the file.
+The host emits these errors because the *\*nix* system has been globally configured to load the *libcontrast_universal_installer.so* library for every binary on the system to determine if it should inject the Java agent into it. Even if you manually removed the library, the global configuration of dynamic loader is still referencing it and is failing/warning when it can't find the file.
+
+### Linux 
 
 On a Linux system, the shell command is still running, even though you see errors about failing to load the *contrast.so* library. Open up the */etc/ld.so.preload* file as root and remove the entry for the contrast libraries.
 
-On a Solaris system, the ld.so process will fail to execute the desired binary if the ld preload library doesn't exist. This causes all commands to be broken. To recover if you still have a shell prompt, use the `LD_NOCONFIG` environment variable to temporarily disable the custom dynamic loader config and remove the config itself. In Solaris, there are two configs that must be removed - one for 32bit and one for 64bit.
+### Solaris 
+
+On a Solaris system, the ld.so process will fail to execute the desired binary if the *ld preload* library doesn't exist. This causes all commands to be broken. 
+
+If you need to recover, and still have a shell prompt, use the `LD_NOCONFIG` environment variable to temporarily disable the custom dynamic loader `config` setting and remove the *config* file itself. In Solaris, there are two `config` settings that must be removed - one for 32-bit and one for 64-bit.
 
 ```
 LD_NOCONFIG=1 sudo LD_NOCONFIG=1 rm /var/ld/ld.config
 
 LD_NOCONFIG=1 sudo LD_NOCONFIG=1 rm /var/ld/sparcv9/ld.config
 ```
 
-There are two `LD_NOCONFIG` settings because sudo is a funny beast. The first is for the `sudo` command itself, the second is for the sudo environment to execute the `rm` command.
+> **Note:** There are two `LD_NOCONFIG` settings because sudo is a funny beast. The first is for the `sudo` command itself, the second is for the sudo environment to execute the `rm` command.
 
 If you're in this situation, and don't have an active shell on Solaris, you will likely need to boot into [recovery mode](https://docs.oracle.com/cd/E36784\_01/html/E39134/glmwr.html) via an installation disk or backup Boot Environment, and remove the stale *ld.config* files from there.
```

Comment:
@gilday The 'recovery mode' link needs to be updated.

Reply:
😩 Sometimes I think Oracle kills their links on purpose to encourage you to buy a support contract, because their links **never** work!